ndcube/ndcube.py

```diff
@@ -584,16 +594,16 @@ def _get_crop_item(self, *points, wcs=None):
                         raise TypeError(f"{type(value)} of component {j} in point {i} is "
                                         f"incompatible with WCS component {comp[j]} "
                                         f"{classes[j]}.")
-            return utils.cube.get_crop_item_from_points(points, wcs, False)
+            return utils.cube.get_crop_item_from_points(points, wcs, False, keepdims)
```

Comment:
:sparkles: pedantry :sparkles: 
```suggestion
            return utils.cube.get_crop_item_from_points(points, wcs, False, keepdims=keepdims)
```